server.js

```diff
@@ -23,12 +25,86 @@ const app = express();
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: "Service unavailable" });
+  }
+});
+
+//import and define the schema
+
+const Book = mongoose.model("Book",
+{
+  bookID: Number,
+  title: String,
+  author: String,
+  average_rating: Number,
+  isbn: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number,
+});
+
+//Reset the database
+/*if (process.env.RESET_DB) {
+  console.log("Resetting database");
+
+  const seedDatabase = async () => {
+    await Book.deleteMany();
+    booksData.forEach((book) => {
+      new Book(book).save();
+    });
+  };
+
+  seedDatabase();
+}*/
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = listEndpoints(app);
```

Comment:
godo to see you used express-list-endpoints. you could even write your code in one line, like this:

app.get("/", (req, res) => {
  res.send(listEndpoints(app));
});